babel.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = api => {
+  // Use only in test environment
```

Comment:
how do we feel about putting this in `package.json`? Same with the `.jestconfig.json`?

Reply:
For the `.jestconfig.json` you can literally just copy what you have and do
```js
"jest" {
  // what you have...
}
```

Reply:
I'm going to merge this for now so not blocking, but would like to discuss :)